releases/release-1.31/release-notes/maps/pr-123398-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 123398
+releasenote:
+  text: The kubelet is no longer able to recover from device manager state file older
```

```suggestion
  text: Kubelet is no longer able to recover from device manager state file older
```

https://github.com/kubernetes/sig-release/pull/2542#discussion_r1653181197